data/ovs/001-ovs-cni.yaml

```diff
@@ -19,12 +19,8 @@ spec:
     spec:
       serviceAccountName: ovs-cni-marker
       hostNetwork: true
-      nodeSelector:
-        beta.kubernetes.io/arch: amd64
-      tolerations:
-        - key: node-role.kubernetes.io/master
-          operator: Exists
-          effect: NoSchedule
+      nodeSelector: {{ toYaml .Placement.NodeSelector | nindent 8 }}
```

Comment:
If I'm not overlooking it, you're missing
```
data.Data["Placement"] = conf.PlacementConfiguration.Workloads
```
in `pkg/network/ovs.go` renderOvs function

Reply:
totally. thanks!